### PR TITLE
Add <SubNav> and <SubNavItem> components

### PR DESCRIPTION
**Pull request by paulocfjunior**
image

### CODE REVIEW

File: src/components/SubNav.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+} from '@modulz/radix-system';
+
+type SubNavProps = MarginProps & PaddingProps & HeightProps & MinHeightProps;
+
+const styleProps = compose(margin, padding, height, minHeight);
+
+export const SubNav = styled('nav')<SubNavProps>(
+  css({
+    height: 48,
+    backgroundColor: 'light',
```
Comment (matthieuh) [inline src/components/SubNav.tsx]: `light` is not a color existing in our theme

**Comment by ldez** [comment]
@paulocfjunior you don't need to "merge master" unless you have conflicts to manage: the bot update the PR before the merge.

FYI to merge a PR, after the PR was approved, you have to add the label `status/3-needs-merge` and the bot will manage the rebase of the PR and the merge.

**Comment by paulocfjunior** [comment]
BTW, @ldez, I've fixed the point that Matthieu mentioned and also changed the `<SubNavItem>` component to have an `<a>` as primitive (instead of a button, as is in `<Nav>`), because I noticed that we usually use it as a link to other routes. Does it make sense?

**Comment by matthieuh** [comment]
> BTW, @ldez, I've fixed the point that Matthieu mentioned and also changed the `<SubNavItem>` component to have an `<a>` as primitive (instead of a button, as is in `<Nav>`), because I noticed that we usually use it as a link to other routes. Does it make sense?

I add the ability to change primitive in the Nav component. Do you think that it could possible to handle it there as well ? https://github.com/containous/faency/pull/4/commits/4e860e9099b38bd232148a9268ffbe5b3f44e1cc

File: src/components/SubNav.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    paddingY: 0,
+    paddingX: 3,
+    textDecoration: 'none',
+    userSelect: 'none',
+    WebkitTapHighlightColor: 'rgba(0, 0, 0, 0)',
+    transition: 'transition: color 0.3s, background-color 0.3s',
+    '&::-moz-focus-inner': {
+      border: 0,
+    },
+    '&:hover': {
```
Comment (matthieuh) [inline src/components/SubNav.tsx]: I found that we miss to set hover styling when the item is active. I handled it in the Nav component https://github.com/containous/faency/pull/4/commits/9c661b8e3759e27c189f7543b7acda452d173902

**Comment by paulocfjunior** [comment]
@matthieuh, it's ready for a final review and approval :heavy_check_mark:

**Pull request merged**